karma.conf.js

```diff
@@ -21,6 +21,12 @@ module.exports = function (config) {
       fixWebpackSourcePaths: true
     },
     reporters: ['progress', 'kjhtml'],
+    customLaunchers: {
+      ChromeHeadlessNoSandbox: {
```

Comment:
karma is a JS unit testing framework, if you're not using it you ought to kill this code :)

Reply:
okay removed and running successfully.